Doc/library/ssl.rst

```diff
@@ -764,6 +769,15 @@ Constants
 
    .. versionadded:: 3.4
 
+.. data:: OP_NO_TLSv1_3
+
+   Prevents a TLSv1.3 connection. This option is only applicable in conjunction
+   with :const:`PROTOCOL_TLS`. It prevents the peers from choosing TLSv1.3 as
+   the protocol version. TLS 1.3 is available with OpenSSL 1.1.1 or later.
+   With older versions, the flag defaults to *0*.
```

Comment:
@pitrou: The advantage of making the flag constants unconditionally available isn't avoiding the labor of writing `getattr`; it's that if you only test your code with one version of OpenSSL, then it'll still probably work on other systems too. IIRC Twisted's latest release crashes on old CentOS, because they were accessing these flags without `getattr` and no-one noticed because no-one tested on the right platform. (They use pyopenssl rather than stdlib ssl, but the same principle applies.)